cmd/index.go

```diff
@@ -68,7 +68,13 @@ func indexFn(cmd *cobra.Command, args []string) {
 	t := time.Now()
 	date := strings.ToLower(t.Format(time.RFC3339))
 
-	err, prevDate := es.ProdAliasedIndexDate(common.ESC)
+	esc, err := common.ESC.GetClient()
+	if esc == nil || err != nil {
+		log.Error(errors.Errorf("No elastic found."))
```

Comment:
You have to add err to the generate log, otherwise we would not know the reason elastic is down:
errors.Wrap(err, "Some additional text to the existing error.")

Make sure to apply this to all places.

Reply:
Can there be a case where esc == nil and err != nil?